.github/workflows/unit-tests-recipes.yml

```diff
@@ -35,13 +35,21 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Get merge-base commit
```

Comment:
We want changed-files to show the files changed in this PR, not necessarily all differences between ToT main and the current branch. The merge queue will make sure that these run at least once where ToT main is the merge base